internal/storage/postgres/vehicle/create.go

```diff
@@ -10,11 +10,16 @@ import (
 
 func defaultVehicle() *entities.Vehicle {
 	return &entities.Vehicle{
-		NumberPlate:   "",
-		Description:   "",
-		WaterCapacity: 0,
-		Type:          entities.VehicleTypeUnknown,
-		Status:        entities.VehicleStatusUnknown,
+		NumberPlate:    "",
+		Description:    "",
+		WaterCapacity:  0,
+		Type:           entities.VehicleTypeUnknown,
+		Status:         entities.VehicleStatusUnknown,
+		Model:          "",
```

Comment:
In your migration you have set the default model type to `Unknown`. Here you set the default to an empty string. If you now create a vehicle with this default type and do not specify a model, the model in the database will be an empty string. Either set the default model here to `Unknown` as in the migration, or remove the default value in the table definition.